### ISSUE TITLE
Add instructions on how to "install" web-design-standards on a new website

### ISSUE THREAD

**Issue by alex** [18F/web-design-standards#870]
Something like: http://getbootstrap.com/getting-started/#download-cdn from the bootstrap getting started instructions which covers standard installation practices like just linking to a CDN or npm/bowser, would go a hugely long way to making it easier to use.

:us:

Thanks!

**Comment by paultag** [comment]
Perhaps using Travis to do a continuous deployment of the assets to S3 would be cool to do, too!

**Comment by mollieru** [comment]
cc @maya

**Comment by maya** [comment]
@alex @paultag we're working on providing more depth instructions for people to use the standards on a new website.

In terms of installing and distribution, we started the npm idea here: [`add-npm`](https://github.com/18F/web-design-standards/tree/add-npm) (we haven't had the bandwidth to complete it yet) and created a ruby gem here [us_web_design_standards_gem](https://github.com/18F/us_web_design_standards_gem).

Let us know what you think.

**Comment by maya** [comment]
Re: CDN - deploying to S3 to provide a hotlink is likely not in scope for alpha/beta, but more possible when we hit 1.0.

**Comment by mollieru** [comment]
@angel

**Comment by maya** [comment]
Related #767.

**Comment by edwelker** [comment]
@maya I have cycles (and motivation) to work on this... what more did you plan for add-npm?

**Comment by maya** [comment]
@edwelker thanks for your note! @jeremiak and @msecret would know more about what more we plan for `add-npm`.  We're pausing for a bit before starting up the next phase of work.

**Comment by edwelker** [comment]
Sure! Let me know if we can help.  Full disclosure: NCBI (www.ncbi.nlm.nih.gov) is going to be using these web-design-standards as the basis of a few new experiments (which may become production), so we've got a vested interest in contributing. :)

**Comment by maya** [comment]
Just updating this that you can install the Draft Standards via npm now.

**Comment by juliaelman** [comment]
@alex thanks for opening this issue! We have updated our `README.md` file to include "Getting Started" instructions and information on how to install via `npm`. 

As for CDV hosting, we've discussed some type of functionality implementing and are working through some possible options. See this comment for more info (and you love to hear your thoughts on security on this too!): https://github.com/18F/web-design-standards/pull/1100#issuecomment-207456638

Closing this issue, as the primary requested functionality has been fulfilled.
